os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -103,6 +104,7 @@
 bool abort_mode = false;
 uint32_t assert_pc;
 extern uint32_t g_assertpc;
+extern mqd_t gbinmgr_mq;
```

Comment:
Can you please give an '_' after 'g' .
gbinmgr_mq  ==> g_binmgr_mq

Reply:
used common fd for both read and write

Reply:
modified